func3d.py

```diff
@@ -90,6 +90,85 @@ def pgradient(self, rvec):
         akderiv = 2*a/(1+a)**3 * r
         return akderiv
 
+    
+class ExpCuspFunction:
+    """
+    :math:`b(r) = \frac{c p(r/r_{cut})}{1+\gamma*p(r/r_{cut})}` 
+    where 
+    :math:`p(y) = y - y^2 + y^3/3`
+    """
+    def __init__(self, c, gamma, rcut):
+        self.parameters={}
+        self.parameters['c'] = c
```

Comment:
For basis functions we do not need an overall scale parameter since they'll be multiplied by a coefficient.